src/engraving/dom/box.cpp

```diff
@@ -431,7 +431,7 @@ void Box::manageExclusionFromParts(bool exclude)
                 continue;
             }
             MeasureBase* newMB = nullptr;
-            if (!excludeFromOtherParts()) {
+            if (!next()->isMeasure() && !next()->excludeFromOtherParts()) {
                 for (auto e : linkList()) {
```

Comment:
Shouldn't this be `next()->linkList()` as well? (assuming that `newMB` is supposed to be the new `next()` for the new box that will be created)

Reply:
Yes, of course, thanks.

Reply:
@cbjeukendrup I was also thinking, if not to create new function `MeasureBase::getInScore(Score*, bool useNextMeasureFallback")` which returns MeasureBase of given score.

Than I could replace all that new code by `MeasureBase* newMB = next().getInScore(score, true);`

What do You think?

Reply:
That sounds like a good idea!